content/en/about/supported-releases/index.md

```diff
@@ -11,12 +11,12 @@ changes. For more information refer to the [Istio support policy](../release-cad
 
 ## Support status of Istio releases
 
-| Version         | Currently Supported   | Release Date    | End of Life       |
-|-----------------|-----------------------|-----------------|-------------------|
-| master          | No, development only  |                 |                   |
-| 1.7             | Yes                   | August 21, 2020 |                   |
-| 1.6             | Yes                   | May 21, 2020    | November 21, 2020 |
-| 1.5 and earlier | No                    |                 |                   |
+| Version         | Currently Supported   | Release Date    | End of Life       | Supported Kubernetes Versions |
+|-----------------|-----------------------|-----------------|-------------------|-------------------------------|
+| master          | No, development only  |                 |                   |                               |
+| 1.7             | Yes                   | August 21, 2020 |                   | 1.15, 1.16, 1.17, 1.18, 1.19  |
```

Comment:
I think 1.15 is not supported.  Also, neither is 1.19.  Could you double check our docs?

Reply:
Double checked https://istio.io/latest/faq/general/#what-deployment-environment and added 1.8 info